TestControllers/TestController.psm1

```diff
@@ -726,6 +726,9 @@ Class TestController {
 				$executionCount += 1
 				Write-LogInfo "($executionCount/$($this.TotalCaseNum)) testing started: $($currentTestCase.testName)"
 				Write-LogInfo "SetupConfig: { $(ConvertFrom-SetupConfig -SetupConfig $currentTestCase.SetupConfig) }"
+				if ($vmData -and $this.TestPlatform -imatch "HyperV") {
```

Comment:
How about refresh for all platforms? It's better to keep upper level logic consistent.

Reply:
IP won't change in Azure platform.